Python.md

```diff
@@ -15,23 +20,63 @@ As most of the other projects written in Python, we use [PEP 8]
 (https://www.youtube.com/watch?v=wf-BqAjZb8M)" by Raymond Hettinger at PyCon 2015,
 which talks about the famous P versus NP problem.*
 
+If the project has a `pylintrc` file, you can use pylint to check for potential style violations:
+
+    pylint --rcfile=pylintrc
+
 The general idea is to make the code within a project consistent and easy to interpret (for humans).
```

Comment:
I think most of these will become unnecessary if we just had Pylint running in Jenkins and making sure that code is formatted correctly. But it seems like a good idea to have a "Common style issues" section pointing these out again, especially considering that we don't have Pylint set up everywhere.